src/main/java/Main.java

```diff
@@ -1,8 +1,22 @@
-public class Main {
+import java.io.IOException;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
 
+public class Main {
 
     public static void main(String[] args) {
-
+        String fileName = "numbers.txt";
+        NumbersReader reader = new NumbersReader();
+        try {
+            Map<Integer, List<Integer>> numbers = reader.readFile(fileName);
+            Collection<List<Integer>> values = numbers.values();
+            for (List<Integer> value : values) {
```

Comment:
Jeżeli zmienisz typ mapy na <Integer, Integer>, to tutaj najlepiej będzie iterować po kluczach, albo `entrySet()`